apps/stirling-pdf/0.36.4/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.36.4
+    image: frooodle/s-pdf:0.36.5
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is an update from the previous version of the Docker image for `stirling-pdf` from version 0.36.4 to 0.36.5. No other changes have been made to the configuration in this diff, so no specific issues, optimizations, or irregularities were found. The service definition remains consistent with the previous setup.